<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Star Typer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="Output"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>U</string>
          </property>
          <property name="buddy">
           <cstring>Ubox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Ubox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>B</string>
          </property>
          <property name="buddy">
           <cstring>Bbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Bbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>V</string>
          </property>
          <property name="buddy">
           <cstring>Vbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Vbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>R</string>
          </property>
          <property name="buddy">
           <cstring>Rbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Rbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>I</string>
          </property>
          <property name="buddy">
           <cstring>Ibox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Ibox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>J</string>
          </property>
          <property name="buddy">
           <cstring>Jbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Jbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>H</string>
          </property>
          <property name="buddy">
           <cstring>Hbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Hbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>K</string>
          </property>
          <property name="buddy">
           <cstring>Kbox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Kbox">
          <property name="value">
           <double>25.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>dU</string>
           </property>
           <property name="buddy">
            <cstring>Ubox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Ubox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>dB</string>
           </property>
           <property name="buddy">
            <cstring>Bbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Bbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>dV</string>
           </property>
           <property name="buddy">
            <cstring>Vbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Vbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>dR</string>
           </property>
           <property name="buddy">
            <cstring>Rbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Rbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>dI</string>
           </property>
           <property name="buddy">
            <cstring>Ibox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Ibox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>dJ</string>
           </property>
           <property name="buddy">
            <cstring>Jbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Jbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>dH</string>
           </property>
           <property name="buddy">
            <cstring>Hbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Hbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>dK</string>
           </property>
           <property name="buddy">
            <cstring>Kbox</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="Kbox_2">
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="TeffBox">
          <item>
           <property name="text">
            <string>Teff - free</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="logZBox">
          <item>
           <property name="text">
            <string>log Z - free</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="loggBox">
          <item>
           <property name="text">
            <string>log g - free</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="EbvBox">
          <item>
           <property name="text">
            <string>E(B-V) - free</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="plotButton">
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="text">
           <string>Load Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Ubox</tabstop>
  <tabstop>Bbox</tabstop>
  <tabstop>Vbox</tabstop>
  <tabstop>Rbox</tabstop>
  <tabstop>Ibox</tabstop>
  <tabstop>Jbox</tabstop>
  <tabstop>Hbox</tabstop>
  <tabstop>Kbox</tabstop>
  <tabstop>calculateButton</tabstop>
  <tabstop>plotButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>loadButton</tabstop>
  <tabstop>Output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
